modules/client-configuration/pages/configuration-basics.adoc

```diff
@@ -0,0 +1,19 @@
+[[configuration.basics]]
+= Configuration Basics
+
+{productname} requires a number of steps to prepare the environment for clients registration before a wide range of its operations can be utilised.
+
+This section contains summary of the initial configuration steps that are necessary to support environment operations following successful {productname} installation and setting up.
+
+
+ifeval::[{suma-content} == true]
+For more information about installing {productname}, see xref:installation:install-server-unified.adoc[].
+For more information about setting up {productname}, see xref:installation:server-setup.adoc[].
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+For more information about installing {productname}, see xref:installation:install-uyuni.adoc[].
+For more information about setting up {productname}, see xref:installation:uyuni-server-setup.adoc[].
+endif::[]
+
+For a detailed decription of further configuration steps, see the following pages.
```

Comment:
We'd better avoid statements such as "See the following pages".  Instead we want proper xrefs.